tests/validation_tests/test_pre_checks.py

```diff
@@ -105,11 +109,47 @@ def valid_workbook():
         ),
         "8 - Review & Sign-Off": pd.DataFrame(
             {
-                0: ["test", "", "", "", "", "", "", "", ""],
-                1: ["", "", "", "", "", "", "", "", ""],
-                2: ["", "", "", "", "", "", "", "", ""],
-                3: ["", "", "", "", "", "", "", "", ""],
-                4: ["", "", "", "", "", "", "", "", ""],
+                0: ["", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""],
```

Comment:
Not that you necessarily need to implement this here, but did you know an equivalent way to write a list of repeating values in python is, for example: `[""] * n`

Reply:
Ah yes, I did, but it completely slipped my mind whilst mocking these. Will be a lot easier writing as above in the future, thanks